test/e2e/utils/kubernetes.go

```diff
@@ -199,10 +207,9 @@ func (k TestKubernetes) CleanNamespaceAndLogWithPanic(t *testing.T, namespace st
 	}
 }
 
-func (k TestKubernetes) PrintAllResources(namespace string, list runtime.Object, set labels.Set) {
-	if err := k.Kubernetes.ResourcesList(namespace, set, list, context.TODO()); err != nil {
-		LogError(err)
-	}
+func (k TestKubernetes) PrintAllResources(dir, namespace, suffix string, list runtime.Object, set labels.Set) {
```

Comment:
We should rename this method to reflect the fact we're no longer printing the resources but writing them to a file. Something like `WriteAllResourcesToFile`